Cosmetic changes

emptynick opened the pull request:
This PR is meant for _small_ non-breaking changes like

- Hard coded strings instead of translations
- Wrong names in Labels

Generally things that are not really worth a dedicated PR.
If you know of any things like this, let me know.

**Changes made:**

- The color from Menu-Builder is now applied to Menu-Items
- Replaced a hard coded string with a translation
- Changed a few Label `for` attributes
- Translated the whole `Compass` section
- Removed a file, whose reference was removed (See #1868)

Fixes #1707
Fixes #1868

Commit:
Small fixes

Commit:
Remove original string

MartynasJanu commented on resources/views/settings/index.blade.php:
```diff
@@ -366,7 +366,7 @@
                     </div>
                     <div class="col-md-3">
-                        <label for="asdf">{{ __('voyager::voyager.generic.type') }}</label>
+                        <label for="type">{{ __('voyager::voyager.generic.type') }}</label>
```
Shouldn't the <select> have the id="type" property? As far as I can see without context, the "for" attribute has no meaning

emptynick replied:
Yes, it could be removed completely, but for readability I would just keep it with a _real_ name

Commit:
more labels

Commit:
Remove left-over file

It was introduced here:
https://github.com/the-control-group/voyager/commit/078a7c81ca382fc1609d0e97b6b2857d2a0533c2#diff-36bd69ef7ba5df2912a70baf254a067bR207
And removed:
https://github.com/the-control-group/voyager/commit/a5dcc7c23ad65dcf0b2206b22f48e19f2664eee0#diff-36bd69ef7ba5df2912a70baf254a067bL182
After 6 days.

Commit:
Menu-Item color is now applied

Commit:
Only set color-prop if color != black

Commit:
Fix 1707

Commit:
Replaced hard coded string in database

Commit:
Hard coded string in Posts

Commit:
Hard coded strings in Media-Manager

Commit:
Search string in BREAD-Browse

Commit:
Download link in BREAD-Read

Commit:
Fix usage

Commit:
Translate compass

Commit:
Make StyleCI happy

Commit:
Compass Datatable translated

fletch3555 committed:
Merge branch '1.x-dev' into cosmetics
